geoh5py/objects/object_base.py

```diff
@@ -181,20 +181,18 @@ def add_data_to_group(
 
         :return: The target property group.
         """
-        if isinstance(data, (Data, uuid.UUID)):
+        if isinstance(data, Data | uuid.UUID | str):
             data = [data]
 
         if isinstance(property_group, str):
             associations = []
```

Comment:
All of this mecanics are duplicates with the new property_group

Reply:
Could be, yes. I will let you modify on your side too.